concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1165,7 +1165,7 @@ jobs:
               DISABLE_USER_CREATION: {{disable_user_creation}}
               OAUTH_CLIENT_ID: {{oauth_client_id}}
               OAUTH_CLIENT_SECRET: {{oauth_client_secret}}
-              COMPOSE_ACCESS_TOKEN: {{compose_access_token}}
+              COMPOSE_API_KEY: {{compose_api_key}}
```

Comment:
Unrelated to this story: Why is this passed to this task? I don't see where it is actually used in the manifest.